src/coreclr/binder/assemblybindercommon.cpp

```diff
@@ -107,53 +107,6 @@ namespace BINDER_SPACE
             return true;
         }
 
-        const WCHAR* s_httpURLPrefix = W("http://");
-        HRESULT URLToFullPath(PathString &assemblyPath)
-        {
-            HRESULT hr = S_OK;
-
-            SString::Iterator pos = assemblyPath.Begin();
-            if (assemblyPath.MatchCaseInsensitive(pos, s_httpURLPrefix))
-            {
-                // HTTP downloads are unsupported
-                hr = FUSION_E_CODE_DOWNLOAD_DISABLED;
```

Comment:
I think we can remove the `FUSION_E_CODE_DOWNLOAD_DISABLED` error code and message too.

Reply:
This and other similar no-longer-used HRESULTs are coupled with the managed exceptions today. It would be nice to clean them up, but I am not going to do in this PR.